### DISCUSSION TITLE
Can we run ARC on EKS fargate node ?

### DISCUSSION THREAD

**Discussion by SejalParmar8796** [actions/actions-runner-controller#1675] [Questions]
Hi,
I have one EKS cluster and the fargate node inside that. I tried creating the ARC in fargate node. But in "kubectl apply" it gives the error. The error screenshot is attached below. So, just want to confirm whether we can run ARC in fargate or not.
![image](https://user-images.githubusercontent.com/57346897/181438771-e5d7a094-7a4f-4a48-9e47-f6b34bb583a9.png)

**Comment by toast-gear** [comment]
https://github.com/actions-runner-controller/actions-runner-controller/issues/631 we've had reports it's possible but no one has provided a `show and tell` in dicussions for the specifics
